docs/internals/security.txt

```diff
@@ -26,10 +26,20 @@ implications, please send a description of the issue via email to
 ``security@djangoproject.com``. Mail sent to that address reaches the `security
 team <https://www.djangoproject.com/foundation/teams/#security-team>`_.
 
+.. note::
+
+    When reporting a potential security vulnerability, it is important that
+    your code follows security best practices. Users have a responsibility to
+    ensure their code adheres to these standards, as Django cannot mitigate
+    vulnerabilities introduced by user errors (a common example being the failure
+    to sanitize user input), other examples can be found here :ref:`security
+    topics <cross-site-scripting>`.
```

Comment:
Small grammatical tidy-ups - no need to parenthesize when a new sentence works well.

 ```suggestion
    When reporting a potential security vulnerability, it is important that
    your code follows security best practices. Users have a responsibility to
    ensure their code adheres to general standards, as Django cannot mitigate
    vulnerabilities introduced by user code. A common example is the failure
    to sanitize user input. For other examples, see the :ref:`security
    topics <cross-site-scripting>`.
```